WebDriverTests/TestExpectations.json

```diff
@@ -2031,7 +2039,7 @@
                 "expected": { "all": { "status": ["PASS"]}}
             },
             "test_create_remove_contexts": {
-                "expected": { "all": { "status": ["FAIL"], "bug": "webkit.org/b/288107"}}
+                "expected": { "all": { "status": ["FAIL"], "bug": "webkit.org/b/288107", "note": "False PASS, Error on the teardown fixture, given we don't support user contexts."}}
```

Comment:
Seems like we need a separate gardening commit just to update user context related results?

Reply:
All these gardening entries are related to this subscription ID patch, even if not under `subscribe/` or `unsubscribe`. Usually they were tests that either:

- Check if a given event was not emitted: Most of there are now passing, as `session.subscribe` return the subscription ID.
- Check if a given event was emitted: These now are timeout instead of fail, as we likely don't implement the event yet.

Regarding this specific entry, the test body itself is passing, thus the "unexpected pass" result, but if I mark as  `["PASS"]` or `["FAIL", "PASS"]`, the teardown step fails and reports an error.  I'll try to run the WPT tests directly (outside webkit) to see if it's something on the fixtures or on our tooling wrapping the expectations.